<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RV32I Emulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="5" colspan="3">
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Register">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTableView" name="RegisterList">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>380</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>380</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Memory">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="0">
         <widget class="QTableView" name="MemoryList">
          <property name="minimumSize">
           <size>
            <width>380</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>380</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>50</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="registerLabel">
             <property name="text">
              <string>Register</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="gotoRegister"/>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="sectionLabel">
             <property name="text">
              <string>Section</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="gotoSection"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0" rowspan="4">
     <widget class="QPlainTextEdit" name="textBrowser">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QPushButton" name="DeleteConsole">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="application.qrc">
        <normaloff>:/Images/Delete.png</normaloff>:/Images/Delete.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="5">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Console</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5" colspan="3">
     <widget class="QPlainTextEdit" name="Console">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>150</height>
       </size>
      </property>
      <property name="plainText">
       <string>test: </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_File"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionVersion"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionReset"/>
   <addaction name="actionStep"/>
   <addaction name="actionRun"/>
   <addaction name="actionEnd"/>
   <addaction name="actionStop"/>
  </widget>
  <action name="actionLoad_File">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load File</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="text">
    <string>Version</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/Images/Reset.png</normaloff>:/Images/Reset.png</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionStep">
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/Images/Step.png</normaloff>:/Images/Step.png</iconset>
   </property>
   <property name="text">
    <string>Step</string>
   </property>
  </action>
  <action name="actionEnd">
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/Images/End.png</normaloff>:/Images/End.png</iconset>
   </property>
   <property name="text">
    <string>End</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/Images/Stop.png</normaloff>:/Images/Stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/Images/Run.png</normaloff>:/Images/Run.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>textBrowser</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>Console</tabstop>
  <tabstop>DeleteConsole</tabstop>
 </tabstops>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections/>
</ui>
